simulator/src/main/java/simblock/simulator/Main.java

```diff
@@ -347,6 +349,52 @@ public static void constructNetworkWithAllNodes(int numNodes) {
     getSimulatedNodes().get(0).genesisBlock();
   }
 
+  public static void constructNetworkWithFarmers(int numFarmers) {
```

Comment:
probably can combine with constructNetworkWithNodes method and just separate the differences